NEXT_CHANGES.rst

```diff
@@ -19,6 +19,14 @@ ixdat 0.2.7
 API changes
 -----------
 
+readers
+^^^^^^^
+- ``BiologicMPRReader`` to read biologic .mpr files using the ``eclabfiles`` package.
+  That package is not added as a requirement - instead if the user tries to read the
+  file without having it installed, they are encouraged to export .mpt instead.
```

Comment:
But I do want to discuourage it! People should use .mpt, because then I can debug it if it goes wrong. MPR is only for when the MPT's are forgotten or broken.

Reply:
I've now made this more explicit by raising a warning each time the MPR reader is used.